lib/GoogleAutocomplete.tsx

```diff
@@ -39,7 +39,12 @@ export interface DefaultProps {
   minLength: number;
   debounce: number;
   language: string;
-  queryTypes: 'address' | 'geocode' | 'cities' | 'establishment' | 'geocode|establishment';
+  queryTypes:
```

Comment:
I dont think this change is need can you rollback this one

Reply:
Will work on submitting an updated PR this weekend

Reply:
Perfect thank you :)